startDevelopmentStack.sh

```diff
@@ -1,5 +1,35 @@
 #!/usr/bin/env bash
 
+local_frontend=false
+
+while getopts ':lh' opt; do
+  case "$opt" in
+    l)
+      echo "Launching in local frontend mode."
+      local_frontend=true
+      export FRONTEND_LOCATION_CONFIG="Localhost"
+      ;;
+
+    h)
+      echo "Usage: $(basename $0) [-l] [-h]"
+      echo "  -l: Run in local frontend mode. Do not launch a frontend docker container. Instead, redirect frontend traffic to localhost."
```

Comment:
Maybe add the info that you have to start the frontend locally with npm run dev, so that is somewhere documented.

Reply:
A great idea. I even went a little bit further and added an extra error page to the proxy for exactly this use-case. That page gets displayed when the frontend is not reachable when you are running in local frontend mode. Feel free to check it out

Reply:
I like the idea! Will check it out